devices.go

```diff
@@ -59,17 +60,6 @@ type Device struct {
 	CustomData          map[string]interface{} `json:"customdata,omitempty"`
 }
 
-type ProvisionEvent struct {
-	ID            string     `json:"id"`
-	Body          string     `json:"body"`
-	CreatedAt     *Timestamp `json:"created_at,omitempty"`
```

Comment:
It seems that sometimes the timestamps in the events are `null`, for example 
```
            "body": "Connected to magic install system",
            "created_at": null,
            "id": null,
            "interpolated": "Connected to magic install system",
            "ip": null,
            "modified_by": null,
            "relationships": [],
            "state": null,
            "type": "provisioning.104"
        },
```

The packngo timestampt handling can not deal with these for some reason. when I run the `TestAccListDeviceEvents` I get:
```
devices_test.go:560: parsing time "null" as ""2006-01-02T15:04:05Z07:00"": cannot parse "null" as """
```


Maybe modifying the Timestamp handling in `timestamp.go` to this would work:

```
// UnmarshalJSON implements the json.Unmarshaler interface.
// Time is expected in RFC3339 or Unix format.
func (t *Timestamp) UnmarshalJSON(data []byte) (err error) {
	str := string(data)
	if str == "null" {
		t = nil
		return
	}
	i, err := strconv.ParseInt(str, 10, 64)
	if err == nil {
		t.Time = time.Unix(i, 0)
	} else {
		t.Time, err = time.Parse(`"`+time.RFC3339+`"`, str)
	}
	return
}
```

Reply:
Let's have it as a pointer, like it was before.

Reply:
Yes, it has to be a pointer. But in case the Packet API returns `null` in the datetime fields, the current function `func (t *Timestamp) UnmarshalJSON(data []byte) (err error) {` will try to parse the string `"null",` which will blow up - `"null"` doesn't pass as int nor RFC3339 datetime string.

^ that happened to me yesterday, I can't replicate it today.

If it happens again, I will fix it in separate PR. You can forget about this in this PR.